3-0-spring-mvc/src/main/java/com/bobocode/mvc/SpringMVC.java

```diff
@@ -0,0 +1,11 @@
+package com.bobocode.mvc;
```

Comment:
@erneman1 first of all – let's fix the location. `3-0-spring-mvc` is a parent module for all Spring MVC exercises. Instead of directly adding everything, you should create a nested module as we do for instance in [here](https://github.com/bobocode-projects/java-fundamentals-course/tree/main/2-0-data-structures-and-algorithms)

Inside `3-0-spring-mvc` module, please create a new module called `3-0-1-hello-spring-mvc` and put your exercise there. The packege `com.bobocode.mvc` is good 👍